push/tasks.py

```diff
@@ -10,17 +10,18 @@
 
 @shared_task
 def task_send_web_push(notification_id):
-    active_subscriptions = GetActiveSubscriptions.get_active_subscription()
-    notification_object = GetNotification.get_notification_by_id(notification_id=notification_id)
+    send_web_push = SendWebPushService(notification_id=notification_id)
+    active_subscriptions = SubscriptionService.get_active_subscription()
     try:
-        SendWebPush(
-            subscription_objects=active_subscriptions,
-            notification_data=notification_object
-        ).send_web_push_to_all_subscribers()
-        notification_object.status = COMPLETED
+        for subscription in active_subscriptions:
+            send_web_push.send_web_push(
+                endpoint=subscription.endpoint,
+                public_key=subscription.public_key,
+                auth_key=subscription.auth_key,
+            )
+        NotificationService.set_notification_status(notification_id=notification_id, status=COMPLETED)
```

Comment:
create method for completing notification status and failing notification status in the notification service and call those here